.app/postcss.js

```diff
@@ -27,8 +28,16 @@ modules.forEach(function(module) {
             console.log(`${alerts.error} ${err}`);
           } else {
             console.log(`${alerts.success} PostCss processed ${module.outDir}${module.outFile}`);
+            if(index == modules.length-1){
```

Comment:
It would be nice if these notifiers were optional... perhaps if we pass a flag `--notify` to the start command or `NOTIFY=true`.